etc/openstack_deploy/group_vars/all/release.yml

```diff
@@ -16,5 +16,12 @@
 # The release tag to use
 # Note that this is set here so that the openstack_release override in osa_defaults
 # is able to use it.
-rpc_release: "r15.0.0"
+rpc_release: "r16.0.0"
```

Comment:
I'd just like an ack that 16 is supposed to be pike (and that 15 was ocata).

Reply:
++ 16 == pike, 15 == ocata.